apiserver/backup.go

```diff
@@ -19,8 +19,12 @@ import (
 	"github.com/juju/juju/state/backups"
 )
 
-var newBackups = func(st *state.State) (backups.Backups, io.Closer) {
-	stor := backups.NewStorage(st)
+var newBackups = func(st *state.State, m *state.Model) (backups.Backups, io.Closer) {
+	backend := struct {
```

Comment:
This is the wrong abstraction here. NewStorage() only needs model UUID, so instead of kludging together a composite struct, just pass in model UUID along with state as parameters to newBackups() and NewStorage()